LocalSettings.php

```diff
@@ -810,7 +748,76 @@
 		'batmanwiki' => true,
 		'snapwikiwiki' => true,
 	],
-
+	'wgCosmosFooterBackgroundColor' => [
+		'default' => '#c0c0c0',
+	],
+	'wgCosmosRailBlacklistedNamespaces' => [
```

Comment:
Defaults are not set in LS for configs in ManageWikiNamespaces.

Reply:
Removed, thanks!